### PR TITLE
Fix block name and structure on order summary

### PR DESCRIPTION
**Pull request by iqit-commerce**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.7.1.x
| Description?  | Due wrong block name and structure, ordered products was not showed correctly in final summary
| Type?         | bug fix 
| Category?     | FO
| BC breaks?    | no
| Deprecations? | /no
| Fixed ticket? | 
| How to test?  | In backoffice shop parametrs > order > Enable final summary then create order go to FO add products, go to checkout, go to payment step

### CODE REVIEW

**Comment by xBorderie** [comment]
Thank you for your contribution!

Sadly, you now have malformed commits instead of just one commit:
* two commits which follow the PS naming convention,
* one Merge commit -- which if merged can break the repository's history!

Could you try and [squash those commits into just one](http://blog.steveklabnik.com/posts/2012-11-08-how-to-squash-commits-in-a-github-pull-request)? [Another explanation of squashing](https://ariejan.net/2011/07/05/git-squash-your-latests-commits-into-one/).

Thank you!

File: themes/classic/templates/checkout/_partials/order-confirmation-table.tpl [hunk 1]
```diff
@@ -22,11 +22,10 @@
  * @license   http://opensource.org/licenses/osl-3.0.php Open Software License (OSL 3.0)
  * International Registered Trademark & Property of PrestaShop SA
  *}
-<div id="order-items" class="col-md-8">
-
-  {block name='order_items_table_head'}
-    <h3 class="card-title h3">{l s='Order items' d='Shop.Theme.Checkout'}</h3>
-  {/block}
+{block name='order_items_table_head'} 
+  <div id="order-items" class="col-md-8">
```
Comment (PrestaEdit) [inline themes/classic/templates/checkout/_partials/order-confirmation-table.tpl]: A <div> without a end tag inside a block ? I think it's clearly not a good idea...

File: themes/classic/templates/checkout/_partials/order-final-summary-table.tpl [hunk 2]
```diff
@@ -34,5 +34,4 @@
     {/if}
   	<a href="{url entity=cart params=['action' => 'show']}"><span class="step-edit"><i class="material-icons edit">mode_edit</i> edit</span></a>
   </h3>
-</div>
```
Comment (PrestaEdit) [inline themes/classic/templates/checkout/_partials/order-final-summary-table.tpl]: End div dropped ?

**Comment by xBorderie** [comment]
Hey @iqit-commerce! Could you update your commits the way I asked you, and answer @PrestaEdit questions? Thank you!

**Comment by maximebiloe** [comment]
As you didn't update tour PR according to the comments, I close this one.

Regards

**Pull request closed without merging**